### DISCUSSION TITLE
Which DB would you recommend for deployment?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12885] [Archive]
With a development in progress using SQLite ( which I found easy to use ), for deployment, i'm thinking to migrate to MariaDB or PostgreSQL. Which one (or other one) DB would you recommend?

**Comment by rijkvanzanten** [comment]
It's mostly personal preference! If you have no particular needs for any specific feature within the database, or other business requirements to choose one over the other, I would personally probably go with Postgres, as that seems to be the most commonly used nowadays. That being said, MySQL has a huge track record and has powered websites for decades, so you can't really go wrong with that either 👍🏻
